draft-ietf-masque-h3-datagram.md

```diff
@@ -103,6 +103,26 @@ hop, but the context ID will remain the same. Context IDs are opaque to
 intermediaries.
 
 
+## Datagram Formats {#datagram-formats}
+
+When an endpoint registers a datagram context (or the lack of contexts), it
+communicates the format (i.e., the semantics and encoding) of datagrams sent
+using this context. This is acccomplished by sending a Datagram Format Type as
+part of the registration capsule, see {{register-capsule}} and
+{{register-no-context-capsule}}. This type identifier is registered with IANA
+(see {{iana-format-types}}) and allows applications that use HTTP Datagrams to
+indicate what the content of datagrams are. Registration capsules carry a
```

Comment:
I read this sentence a few times and still how minor difficulty parsing it (in the context of the surrounding sentences). I don't have an immediate suggestion to improve readability though.

Looking at the example below, it seems this PR is proposing that the nature of DATAGRAM payload **always** has to be communicated by the type. That didn't used to be the case.

Reply:
That's right, the idea would be to now always have an explicit type